src/spec/fetchstock.spec.coffee

```diff
@@ -56,3 +56,11 @@ describe 'FetchStocks', ->
       expect(result.body.results[0].quantityOnStock).toEqual 6
       done()
     .catch done
+
+  it 'should sort by id if it is not overrided', (done) ->
+    sortingParam = ''
+    @fetchStocks.client.inventoryEntries.sort = (path) ->
```

Comment:
You're modifying the function here permanently this causes other tests to fail if run earlier. Please use a spy.